### PR TITLE
newPull

### PR DESCRIPTION
**Pull request by mrfyda**

### CODE REVIEW

File: Test.scala [hunk 1]
```diff
@@ -21,3 +21,11 @@ object Test2 {}
 class Cla3 {}
 
 class SUP {}
+
+class newPull {
+  
+  def coke = ???
```
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala [hunk 2]
```diff
@@ -21,3 +21,11 @@ object Test2 {}
 class Cla3 {}
 
 class SUP {}
+
+class newPull {
+  
+  def coke = ??? 
+  
+  def cola = Some(2).get
```
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Codacy Logo Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Codacy Logo Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: Codacy Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: Codacy Usage of ??? operator

Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/p/2/f/17/issues/source?bid=17&fileBranchId=3507#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/p/2/f/17/issues/source?bid=17&fileBranchId=3507#l29) Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/p/2/f/3507/issues/source?bid=12&fileBranchId=17#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/p/2/f/3507/issues/source?bid=12&fileBranchId=17#l29) Usage of get on optional type.

Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/file/rafael_4/issues/source?bid=ScalaTests&fileBranchId=3#l29) Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/file/rafael_4/issues/source?bid=ScalaTests&fileBranchId=3#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/file/rafael_4/issues/source?bid=ScalaTests&fileBranchId=3#l29) Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/%E2%95%AF%C2%B0%E2%96%A1%C2%B0%E2%95%AF%EF%B8%B5%20%E2%94%BB%E2%94%81%E2%94%BB/file/rafael_4/issues/source?bid=ScalaTests&fileBranchId=3#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l29) Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/p/1/f/3/issues/source?bid=1&fileBranchId=6#l29) Usage of get on optional type.

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/p/1/f/3/issues/source?bid=1&fileBranchId=6#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/330634168/issues/source?bid=2115350&fileBranchId=2144956#l29) Usage of get on optional type.

File: Test.scala [hunk 3]
```diff
@@ -21,3 +21,11 @@ object Test2 {}
 class Cla3 {}
 
 class SUP {}
+
+class newPull {
```
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/330634168/issues/source?bid=2115350&fileBranchId=2144956#l25) Class name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/330634168/issues/source?bid=2115350&fileBranchId=2144956#l27) Usage of ??? operator

File: Test.scala (hunk 2 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/330634168/issues/source?bid=2115350&fileBranchId=2144956#l29) Usage of get on optional type.

File: Test.scala (hunk 3 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/330634168/issues/source?bid=2115350&fileBranchId=2144956#l25) Class name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'

File: Test.scala (hunk 1 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/330634168/issues/source?bid=2115350&fileBranchId=2144956#l27) Usage of ??? operator

File: Test.scala [hunk 4]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
...
+  
+  def coke = ??? 
+  
+  def cola = Some(2).get
```
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/393172617/issues/source?bid=2115350&fileBranchId=2144956#l31) Usage of get on optional type.

File: Test.scala [hunk 5]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
 class Cla3 {}
 
 class SUP {}
+
+class newPull {
```
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/393172617/issues/source?bid=2115350&fileBranchId=2144956#l27) Class name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'

File: Test.scala [hunk 6]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
...
+  
+  def coke = ???
```
Comment (mrfyda) [inline Test.scala]: [Codacy](https://www.codacy.com/public/mrfyda/ScalaTests/file/393172617/issues/source?bid=2115350&fileBranchId=2144956#l29) Usage of ??? operator

File: Test.scala [hunk 7]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
 class Cla3 {}
 
 class SUP {}
+
+class newPull {
+  
+  def coke = ??? 
+
```
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l31) Usage of get on optional type.

Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l31) Usage of get on optional type.

File: Test.scala [hunk 8]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
 class Cla3 {}
 
 class SUP {}
+
+class newPull {
+
```
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l29) Usage of ??? operator

File: Test.scala (hunk 7 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l31) Usage of get on optional type. offset:Some(17)

File: Test.scala (hunk 8 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l29) Usage of ??? operator offset:Some(15)

File: Test.scala [hunk 9]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
 class Cla3 {}
```
Comment (mrfyda) [inline Test.scala]: 23

File: Test.scala [hunk 10]
```diff
@@ -21,3 +21,11 @@ object Test2 {}
...
```
Comment (jasonparser) [inline Test.scala]: 1

File: Test.scala [hunk 11]
```diff
@@ -21,3 +21,11 @@ object Test2 {}
...
```
Comment (jasonparser) [inline Test.scala]: 2

File: Test.scala [hunk 12]
```diff
@@ -21,3 +21,11 @@ object Test2 {}
...
 
 class SUP {}
+
+class newPull {
+  
+  def coke = ??? 
+  
+  def cola = Some(2).get
+  
+}
```
Comment (jasonparser) [inline Test.scala]: 11

File: Test.scala [hunk 13]
```diff
@@ -14,6 +14,8 @@ class Test {
```
Comment (jasonparser) [inline Test.scala]: 1

File: Test.scala [hunk 14]
```diff
@@ -21,3 +23,11 @@ object Test2 {}
```
Comment (jasonparser) [inline Test.scala]: 9

File: Test.scala (hunk 9 above)
Comment (jasonparser) [inline Test.scala]: 10

File: Test.scala (hunk 4 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l31) Usage of get on optional type.

File: Test.scala (hunk 6 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/public/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l29) Usage of ??? operator

File: Test.scala (hunk 4 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/app/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l31) Usage of get on optional type.

File: Test.scala (hunk 6 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/app/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l29) Usage of ??? operator

File: Test.scala (hunk 4 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/app/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l31) Usage of get on optional type.

File: Test.scala (hunk 6 above)
Comment (mrfyda) [inline Test.scala]: [Codacy](http://localhost:9000/app/rafael_4/ScalaTests/file/3/issues/source?bid=1&fileBranchId=6#l29) Usage of ??? operator

File: Test.scala (hunk 4 above)
Comment (mrfyda) [inline Test.scala]: Codacy Issue found: [Usage of get on optional type.](https://www.codacy.com/app/mrfyda/ScalaTests/file/941662032/issues/source?bid=2844648&fileBranchId=2844656#l31)

File: Test.scala (hunk 6 above)
Comment (mrfyda) [inline Test.scala]: Codacy Issue found: [Usage of ??? operator](https://www.codacy.com/app/mrfyda/ScalaTests/file/941662032/issues/source?bid=2844648&fileBranchId=2844656#l29)

**Pull request closed without merging**